float64 timestamp
float64 TEMP_BE1
float64 TEMP_BE2
float64 TEMP_BE3
float64 TEMP_BE4
float64 TEMP_BE5
float64 TEMP_BE6
float64 TEMP_BE7
float64 TEMP_BE8
float64 TEMP_BE9
float64 TEMP_BE10
float64 TEMP_BE11
float64 TEMP_BE12
float64 TEMP_BE13
float64 TEMP_BE14
float64 TEMP_BE15
float64 TEMP_BE16
float64 TEMP_BE17
float64 TEMP_BE18
float64 TEMP_BE19
float64 TEMP_BE20
string from_node
